### PR TITLE
Enable frontier wallet injected connector option

### PR DESCRIPTION
**Pull request by akash-frontier**

### CODE REVIEW

File: packages/connectkit/src/components/Pages/Connectors/index.tsx [hunk 1]
```diff
@@ -81,12 +82,13 @@ const Wallets: React.FC = () => {
...
     const needsInjectedWalletFallback =
-      typeof window !== 'undefined' &&
-      ethereum &&
-      !isMetaMask() &&
-      !isCoinbaseWallet();
+      (typeof window !== 'undefined' &&
+        ethereum &&
+        !isMetaMask() &&
+        !isCoinbaseWallet()) ||
+      isFrontier();
```
Comment (lochie) [inline packages/connectkit/src/components/Pages/Connectors/index.tsx:90]: This check here isn't necessary.

**Review by lochie — CHANGES_REQUESTED** [review]
There's a check in this PR that isn't necessary, I'd suggest removing it.

@akash-frontier could you also merge the latest in from `main`. We will happily merge this in once that's done. Thanks!

**Comment by akash-frontier** [comment]
Hi @lochie, I've resolved the conflicts in the pull request. Please review it when you have the time. Thank you!

**Review by lochie — APPROVED** [review]

**Comment by lochie** [comment]
merged, thanks @akash-frontier!

**Pull request merged**